### PR TITLE
Move blank theme dependencies out of Magento_Theme requirejs-config

### PR DESCRIPTION
**Pull request by mikeoloughlin**
### Description

Creating a new theme that does not fallback to the blank theme causes two files to 404 because Magento_Theme module depends on blank theme assets `js/theme.js` and `js/responsive.js`

### Fixed Issues (if relevant)

1. #3784: js/theme & js/responsive files in wrong requirejs-config file?

### Reproduce bug:

1. Follow the steps to create a new theme from the [devdocs](http://devdocs.magento.com/guides/v2.1/frontend-dev-guide/themes/theme-create.html)
2. Don't define a parent theme:
```
<theme xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Config/etc/theme.xsd">
     <title>New theme</title>
 </theme>
```
3. Clear cache and go to the frontend to see the 404s

### Manual testing steps:
1. New theme should no longer attempt to load `theme.js` or `responsive.js`
2. Switch to blank theme and clear cache. Storefront should load `theme.js` and `responsive.js`

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=8982) <br/>All committers have signed the CLA.

File: app/design/frontend/Magento/blank/requirejs-config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
```
Comment (Igloczek) [inline app/design/frontend/Magento/blank/requirejs-config.js]: I'd say that adding `requirejs-config.js` at theme root should be considered as anti-pattern and Magento core shouldn't promote it, b/c it's against "everything is a module" aproach.

By default this config file is stored in `Theme` module of `Magento` vendor, so following files structure, it should be added as `app/design/frontend/Magento/blank/Magento_Theme/requirejs-config.js` and JS files should be moved from `app/design/frontend/Magento/blank/web/js` to `app/design/frontend/Magento/blank/Magento_Theme/web/js`.

Comment (mikeoloughlin) [reply app/design/frontend/Magento/blank/requirejs-config.js]: That's a good point and I agree with you. I've updated the code and tested it.

Thanks :)

**Review by Igloczek — APPROVED** [review]

**Comment by okorshenko** [comment]
@mikeoloughlin thank you for your contribution!

**Pull request merged**